README.md

```diff
@@ -1,14 +1,42 @@
-API Example Microservice
+Obligations API
 ========================
+The Obligations API allows software packages to:
 
-TODO Describe microservice / API
+
+- retrieve obligations for a user's business income sources
+- retrieve the crystallisation obligations for a user's Income Tax account
+- retrieve the End of Period Statement obligations for a user's business income sources
+
+
+## Requirements
+- Scala 2.12.x
+- Java 8
+- sbt 1.3.13
+- [Service Manager](https://github.com/hmrc/service-manager)
+
+## Run Microservice
+To run the microservice from console, use `sbt run`
```

Comment:
add the default port in here `(starts on port 7793 by default)` at the end of the line